lib/ts/nextjs.ts

```diff
@@ -16,6 +16,7 @@ import { serialize } from "cookie";
 import { errorHandler } from "./framework/express";
 import { CollectingResponse, PreParsedRequest, middleware } from "./framework/custom";
 import { HTTPMethod } from "./types";
+import Session, { SessionContainer, VerifySessionOptions } from "./recipe/session";
 function next(
     request: any,
```

Comment:
add unit tests for this PR - making use of actual nextjs api routes and calling withSession and getSSRSession functions in the test.

Reply:
done